cirq-aqt/cirq_aqt/aqt_sampler.py

```diff
@@ -41,10 +41,9 @@ class AQTSampler(cirq.Sampler):
     """
 
     def __init__(self, remote_host: str, access_token: str):
-        """
-        Args:
-            remote_host: Address of the remote device.
-            access_token: Access token for the remote api.
+        """Args:
```

Comment:
First line should be a description of the method. In this case I think the args should be moved to `Attributes` on the class. https://google.github.io/styleguide/pyguide.html#384-classes